test/libsolidity/syntaxTests/iceRegressionTests/storage_parameter_in_library.sol

```diff
@@ -0,0 +1,8 @@
+library Test {
+    struct Nested { mapping(uint => uint )[2][] a; } uint y;
```

Comment:
This being on a single line initially led me think that the syntax here is much more broken than it really is :)
```suggestion
    struct Nested { mapping(uint => uint )[2][] a; }
    uint y;
```